src/rsz/src/SizeDownMove.cc

```diff
@@ -0,0 +1,326 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2025-2025, The OpenROAD Authors
+
+#include "SizeDownMove.hh"
+
+#include <algorithm>
+#include <cmath>
+#include <string>
+
+#include "BaseMove.hh"
+
+namespace rsz {
+
+using std::pair;
+using std::string;
+using std::vector;
+
+using utl::RSZ;
+
+using sta::ArcDelay;
+using sta::Edge;
+using sta::Instance;
+using sta::InstancePinIterator;
+using sta::LibertyCell;
+using sta::LibertyPort;
+using sta::LoadPinIndexMap;
+using sta::Net;
+using sta::NetConnectedPinIterator;
+using sta::Path;
+using sta::PathExpanded;
+using sta::Pin;
+using sta::Slack;
+using sta::Slew;
+using sta::Vertex;
+using sta::VertexOutEdgeIterator;
+
+bool SizeDownMove::doMove(const Path* drvr_path,
+                          int drvr_index,
+                          Slack drvr_slack,
+                          PathExpanded* expanded,
+                          float setup_slack_margin)
+{
+  Pin* drvr_pin = drvr_path->pin(this);
+  Vertex* drvr_vertex = drvr_path->vertex(sta_);
+  const Path* load_path = expanded->path(drvr_index + 1);
+  Vertex* load_vertex = load_path->vertex(sta_);
+  Pin* load_pin = load_vertex->pin();
+
+  const int fanout = this->fanout(drvr_vertex);
+  if (fanout < 2) {
+    return false;
+  }
```

Comment:
Why not?

Reply:
Good point. I'll check how often this happens, but it may require some different logic since this path is critical and has negative slack. We are only considering paths that have positive slack right now.